frame/collective/src/lib.rs

```diff
@@ -287,6 +306,9 @@ decl_module! {
 	pub struct Module<T: Config<I>, I: Instance=DefaultInstance> for enum Call where origin: <T as frame_system::Config>::Origin {
 		type Error = Error<T, I>;
 
+		const MaxMembers: u32 = T::MaxMembers::get();
```

Comment:
does anyone care about this to put into metadata?

Reply:
i think it would make sense for UI right?

Reply:
yeah probably.